### PR TITLE
Expand the docs around pattern matches in the .bldr.toml files

### PR DESCRIPTION
**Pull request by raskchanky**
Closes https://github.com/habitat-sh/builder/issues/389
Signed-off-by: Josh Black <raskchanky@gmail.com>

### CODE REVIEW

**Comment by raskchanky** [comment]
@thesentinels acceptance

**Comment by thesentinels** [comment]
:metal: I am sending your branch against master to our acceptance environment.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/369242954).

File: www/source/partials/docs/_dev-pkgs-multiple-plans-builder.html.md.erb [hunk 1]
```diff
@@ -12,7 +12,9 @@ To enable this functionality, do the following:
...
+    The `.bldr.toml` file is in TOML format, so create a TOML table specifying the `$pkg_name` value for that plan and then add a `plan_path` field specifying the path to your `plan.sh` file (you do not need to include plan.sh explicitly in the path). If all the files related to the plan are under the plan path, then you are done. Otherwise, you will need an addition 'paths' field specifying Unix-style path globs to files that are associated with the plan you specified in the 'plan_path'. File or directory changes made in these path locations determine which packages will be rebuilt. Basically, when a file is committed, Builder will check to see whether it falls underneath the `plan_path` hierarchy, or matches one of the globs in the `paths` field if it was specified - if the answer is yes, then Builder will issue a build for that commit.
```
Comment (cnunciato) [inline www/source/partials/docs/_dev-pkgs-multiple-plans-builder.html.md.erb]: `need an additional`?

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports acceptance has been deployed](https://travis-ci.org/habitat-sh/habitat/builds/369242954).

I love new and shiny websites!

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports acceptance has been deployed](https://travis-ci.org/habitat-sh/habitat/builds/369253728).

I love new and shiny websites!

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**